.gitignore

```diff
@@ -19,4 +19,10 @@ core/market/decentralized/tests/test-workdir/*
 # Databases
 **/*.db
 **/*.db-shm
-**/*.db-wal
\ No newline at end of file
+**/*.db-wal
+
+# Provider dir
+**/ya-prov
+
+# Requestor dir
+**/ya-req
```

Comment:
please add a newline to the end of the file, as you see the first line of the diff should not have been marked as changed